components/automate-gateway/api/iam/v2/common/tokens.proto

```diff
@@ -4,11 +4,19 @@ package chef.automate.api.iam.v2;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2/common";
 
 message Token {
+    // Unique, optionally user-specified ID. Cannot be changed.
```

Comment:
for v2 I think we always want the user to specify the ID. not sure if these docs are supposed to be written with v2 only in mind

Reply:
yes v2 only.

Reply:
Oh yes, I thought I changed that 🤔